contracts/NFTStakingManager.sol

```diff
@@ -50,9 +60,11 @@ struct ValidationInfo {
   bytes registrationMessage;
   EnumerableSet.Bytes32Set delegationIDs;
   mapping(uint32 epochNumber => uint256 rewards) claimableRewardsPerEpoch; // will get set to zero when claimed
+  EnumerableSet.UintSet claimableEpochNumbers;
```

Comment:
Why do we need this?

Reply:
To iterate through validator rewards per epoch